cluster-provision/gocli/cmd/run.go

```diff
@@ -482,6 +485,16 @@ func run(cmd *cobra.Command, args []string) (retErr error) {
 			nodeQemuArgs = fmt.Sprintf("%s -device vfio-pci,host=%s", nodeQemuArgs, gpuAddress)
 		}
 
+		var vmArgsNvmeDisks []string
+		if len(nvmeDisks) > 0 {
```

Comment:
Would be possible to execute `qemu-img create` command at gocli so we don't touch vm.sh ? maybe adding additional steps at the vmContainerConfig Cmd:.

We want to move as much bash code to gocli as possible.

Reply:
no because the gocli is executed in a separate container and it only sets the command to run in the node container. If we want to remove the script then we need some kind of program that executes the commands inside the node container. The current etup is not different than the way how the script creates the image for block devices (https://github.com/kubevirt/kubevirtci/blob/main/cluster-provision/centos8/scripts/vm.sh#L85)

Reply:
`qemu-img` needs to be executed in the container that provisions the node